### PR TITLE
Using TableManager to get tables.

### PR DESCRIPTION
**Pull request by joakim-hove**
Followup to: https://github.com/OPM/opm-parser/pull/570

### CODE REVIEW

File: opm/core/props/satfunc/SatFuncBase.hpp [hunk 1]
```diff
@@ -117,8 +117,7 @@ namespace Opm
         TableType krow_;
         TableType pcow_;
         TableType krg_;
-        TableType krog_;
-        TableType pcog_;
+        TableType krog_;        TableType pcog_;
```
Comment (atgeirr) [inline opm/core/props/satfunc/SatFuncBase.hpp]: Accidental deletion?

**Comment by atgeirr** [comment]
Looking at this (and its companions) I'd say it looks fine. The above hiccup was the only strange thing I found. Feel free to merge when convenient @joakim-hove, except perhaps we should get in the much-delayed (by me, I know...) #834 first. There might be a little conflict there.

**Comment by andlaus** [comment]
From my side, feel free to merge and I'll rebase #834 if required: I think the conflicts for #834 will be mostly trivial because the satfuncs won't directly use the tables after #834 anymore.

**Comment by atgeirr** [comment]
> From my side, feel free to merge and I'll rebase #834 if required: I think the conflicts for #834 will be mostly trivial because the satfuncs won't directly use the tables after #834 anymore.

OK, @joakim-hove you are clear to merge.

**Comment by joakim-hove** [comment]
Ok Cool - I will!
2. sep. 2015 09:16 skrev "Atgeirr Flø Rasmussen" <notifications@github.com>
følgende:

**Pull request merged**